src/main/java/data/TaskFileReaderWriter.java

```diff
@@ -0,0 +1,163 @@
+package data;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+import utils.Utility;
+
+/**
+ * The TaskFileReaderWriter class provides methods for reading and writing tasks to and from a file.
+ * It includes methods for loading tasks from a file and updating a task file with tasks from a
+ * TaskManager object, and creating a new task file.
+ * @author Nicholas Lee
+ */
+public class TaskFileReaderWriter {
+
+    private static final String DIRECTORY_NAME = "userData";
+    private static final String FILE_NAME = "test.txt";
+
+    public TaskFileReaderWriter() {}
+
+    /**
+     * Creates a new task storage .txt file in the user's drive in a specified directory and file name.
+     *
+     * @return boolean value indicating the success of the file creation.
+     */
+    public boolean createTaskFile() {
```

Comment:
Since this method returns a boolean, you may want to consider using a prefix such as `is`, `has`, `was`. Perhaps something like `canCreateTaskFile` or `hasCreatedTaskFile`?